webpack/webpack.config.prod.js

```diff
@@ -74,11 +70,13 @@ export default {
     new webpack.DefinePlugin({
       '__CLIENT__': true,
       '__PRODUCTION__': true,
-      'process.env.NODE_ENV': JSON.stringify('production'),
-      'process.env.SERVER_PORT': JSON.stringify(serverPort) || JSON.stringify('3000'),
-      'process.env.GRAPHQL_HOST': JSON.stringify(graphQLHost) || JSON.stringify('localhost'),
-      'process.env.GRAPHQL_PROTOCOL': JSON.stringify(graphQLProtocol) || JSON.stringify('http:')
-    })
+      'process.env.NODE_ENV': JSON.stringify('production')
+    }),
+    new webpack.EnvironmentPlugin([
+      'SERVER_PORT',
+      'GRAPHQL_HOST',
+      'GRAPHQL_PROTOCOL'
```

Comment:
you can pass NODE_ENV in here too - wondering if that would make one in webpack.DefinePlugin obsolete?